src/gui/torrentoptionsdialog.cpp

```diff
@@ -63,9 +64,18 @@ TorrentOptionsDialog::TorrentOptionsDialog(QWidget *parent, const QVector<BitTor
     m_ui->setupUi(this);
 
     Q_ASSERT(!torrents.empty());
+    m_ui->savePath->setMode(FileSystemPathEdit::Mode::DirectorySave);
+    m_ui->savePath->setDialogCaption(tr("Choose save path"));
```

Comment:
These 2 lines should move above `Q_ASSERT()`.
Or another way, move `Q_ASSERT()` to the top of this function block.